openstack/nova/values.yaml

```diff
@@ -9,7 +9,9 @@ global:
   novaApiMetadataPortInternal: '8775'
   novaConsolePortPublic: '443'
   master_password: null
-
+  domain_seeds:
+    skip_hcm_domain: false
```

Comment:
Why is this setting added in `global`? IIRC, things in global are supposed to be read by other charts or something - is there another use-case I don't know about or are we striving for that use-case or can this also be non-global?